Project.toml

```diff
@@ -3,7 +3,10 @@ authors = ["Deltares and contributors <ribasim.info@deltares.nl>"]
 description = "Meta-project used to share the Manifest of Ribasim and its dependents"
 
 [deps]
+ReTestItems = "817f1d60-ba6b-4fd5-9520-3cf149f6a823"
+Revise = "295af30f-e4ad-537b-8983-00126c2a3abe"
 Ribasim = "aac5e3d9-0b8f-4d4f-8241-b1a7a9632635"
+TestEnv = "1e6cf692-eddd-4d53-88a5-2d735e33781b"
```

Comment:
I've added these to the main Ribasim project to make it easier to develop.

Reply:
This Project.toml is not the core/Project.toml. Therefore these packages are only loaded when you start the right environment. Nothing changed for the build step. It's to ensure that every developer has the same packages installed. @visr what do you think? Should I remove these lines?

Reply:
Ah, sorry, then this is the dev Project.toml, that should be fine.